app/style/content/preferences/account.less

```diff
@@ -97,8 +96,32 @@
     width: 256px;
   }
 
-  &:focus + .preferences-account-username-label:after {
+  &:focus + .preferences-account-input-icon:after {
     content: "";
     display: none;
   }
 }
+
+.preferences-account-name {
+  font-weight: bold;
+}
+
+.preferences-account-username-wrapper {
+  position: relative;
+  min-height: @line-height-xl * 2;
+}
+
+.preferences-account-username-addsign {
+  position: absolute;
+  top: 0;
+  left: 0;
+  line-height: @line-height-xl;
+  color: @graphite;
+}
+
+.preferences-account-username {
+  padding-left: 22px;
+  &:focus ~ .preferences-account-username-hint {display: block;}
+}
+
+.preferences-account-username-hint {display: none;}
```

Comment:
They are minified by the compilers anyway... and fewer things to think about when deciding when one should be inline or not.. :)